app/main/spawn.js

```diff
@@ -63,23 +66,33 @@ export function startUserAgentService(userAgentClient, options = {}) {
   const profile = options.profiledir || parseArgs().profile;
 
   const NODE = path.join(lib, NODE_CMD);
-  const UA_SERVICE_BIN = path.join(lib, 'services', 'user-agent-service', 'user-agent-service');
+  const UA_SERVICE_BIN = path.join(lib,
+                                   'node_modules',
+                                   'datomish-user-agent-service',
+                                   'bin', 'user-agent-service');
 
-  spawnProcess('UA Service', NODE, [UA_SERVICE_BIN,
+  const child = spawnProcess('UA Service', NODE, [UA_SERVICE_BIN,
     '--port', port,
     '--profile', profile,
     '--version', version,
-    '--content-service-origin', `${endpoints.TOFINO_PROTOCOL}://`,
+    '--content-service-origin', contentServiceOrigin,
   ], {
     detached,
     stdio,
   });
 
   // Connecting to a client immediately is sometimes not necessary,
   // e.g. when starting this service standalone.
+  let connected = null;
   if (userAgentClient) {
-    userAgentClient.connect({ port, host, version });
+    connected = userAgentClient.connect({ port, host, version });
   }
+
+  const stop = () => {
+    child.kill('SIGTERM');
+  };
+
+  return [connected, stop, child];
```

Comment:
This is an unusual return signature for js; as an object `{ connected, stop, child }` would be more idiomatic